build.gradle

```diff
@@ -19,6 +20,9 @@ repositories {
     maven {
         url 'https://repo.eclipse.org/content/repositories/lemminx-releases'
     }
+    maven {
```

Comment:
Usually we only use sonatype during development. Is there a dependency we can only get there or is this a line we'll remove later like `mavenLocal()`?

Reply:
LCLS/lemminx-liberty jars weren't available in the other maven repos, and it failed to download/start without the sonatype repo. I'll check with Matt to see if it's published anywhere else.

Reply:
Ok, I assume open source users can build the dependencies locally if they can't access sonatype.